libs/editor/WordPressEditor/build.gradle

```diff
@@ -31,14 +33,17 @@ apply from: 'https://gist.githubusercontent.com/hypest/f526fe0775dedce0ce0133f14
 
 android {
     compileSdkVersion 28
-    buildToolsVersion '28.0.3'
 
     defaultConfig {
         versionCode 13
         versionName "1.3"
         minSdkVersion 21
         targetSdkVersion 26
```

Comment:
I know it's not part of changes on this PR, but what could be a reason for having `targetSdkVersion` 26 and `compileSdkVersion ` 28? hmm.

Reply:
Updated in f840d763d2310d4be0b8b03cffdf9bc34502a1d9